### PR TITLE
[stable/airflow] revert postgres sub-chart values to match stable/pos…

### PR DESCRIPTION
**Pull request by javefang**
…tgresql 0.13.1

I made a mistake in PR https://github.com/helm/charts/pull/9964/ to wrongly reference
the latest version of stable/postgresql. This commit will fix the compatibility of
this chart with stable/postgresql 0.13.1.

Signed-off-by: Xinghong Fang <xinghong.fang@man.com>

#### What this PR does / why we need it:
Fix postgres sub-chart values to make it compatible with the chart version specified in `requirements.yaml` (0.13.1).

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by javefang** [comment]
Hi @maver1ck , I made a mistake in https://github.com/helm/charts/pull/9964 where I was referencing the latest version of stable/postgresql which made me to use "postgresqlUsername" instead of "postgresUser". This means the head of stable/airflow (0.10.0) is currently broken if people try to use it to deploy. This PR should fix it...

File: stable/airflow/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 0.10.0
```
Comment (gsemet) [inline stable/airflow/Chart.yaml:3]: Sadly this is a impacting change, I recommend to jump to 0.11.0

Comment (javefang) [reply stable/airflow/Chart.yaml:3]: thanks, I have bumped the version

Comment (gsemet) [reply stable/airflow/Chart.yaml:3]: Great :)

**Comment by javefang** [comment]
@gsemet can you give an LGTM, pls?

**Comment by maver1ck** [comment]
Sorry. I'm on vacations :)

**Comment by javefang** [comment]
Ah enjoy! And thank you for taking time to check this :)

**Pull request merged**